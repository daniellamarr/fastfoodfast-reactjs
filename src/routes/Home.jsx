import React from 'react';
import GetMenu from '../components/GetMenu.jsx';

/**
 * Home
 */
export default class Home extends React.Component {
  /**
   * @returns {JSX} jsx
   */
  render() {
    return (
      <div>
        <section id="home">
          <div className="title">
            <h1>Quench The Hunger</h1>
            <h4>Your One Stop Restaurant</h4>
          </div>
        </section>
        <section id="foods">
          <div className="title">
            <h2>Foods</h2><hr className="div" />
          </div>
          <div id="allmenu">
            <GetMenu />
          </div>
          <div className="clear"></div>
        </section>
        <section id="jumbotron">
          <h1>Get it here!</h1>
        </section>
      </div>
    );
  }
}
