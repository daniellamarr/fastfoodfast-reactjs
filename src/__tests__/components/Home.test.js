/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import Home from '../../routes/Home.jsx';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Home />);
});

describe('Loader Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
