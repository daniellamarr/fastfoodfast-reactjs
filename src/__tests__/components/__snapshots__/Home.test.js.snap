// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should match snapshot 1`] = `
<div>
  <section
    id="home"
  >
    <div
      className="title"
    >
      <h1>
        Quench The Hunger
      </h1>
      <h4>
        We are your one stop restaurant offering best meals at affordable prices
      </h4>
      <a
        href="/signup"
      >
        <button>
          Get Started
        </button>
      </a>
    </div>
  </section>
  <section />
  <section
    id="foods"
  >
    <div
      className="title"
    >
      <h2>
        Foods
      </h2>
      <hr
        className="div"
      />
    </div>
    <div
      id="allmenu"
    >
      <Connect(GetMenu) />
    </div>
    <div
      className="clear"
    />
  </section>
  <section
    id="jumbotron"
  >
    <h1>
      Get it here!
    </h1>
  </section>
</div>
`;
